tests/test_simple.py

```diff
@@ -193,3 +193,21 @@ def test_parameterized_select_named_pg(postgresql):
     q.insert_some_value(postgresql)
 
     assert q.get_all_values(postgresql, a=1) == [(1, 2, 3)]
+
+
+def test_without_trailing_semi_colon_pg(postgresql):
```

Comment:
Maybe a simplier and direct test would be to test the generated sql with the ``sql`` attribute.

For instance:

```python

@pytest.mark.parametrize("dbdriver", ("sqlite", "psycopg2"))
def test_without_trailing_semi_colon_pg(dbdriver):
    _queries = (
        "-- name: query\n"
        "SELECT a, b, c FROM foo WHERE a = :a\n")
    q = anosql.from_str(_queries, "psycopg2")
    assert q.query.sql == 'SELECT a, b, c FROM foo WHERE a = %(a)s'
```

Otherwise the PR LGTM, thanks!

Reply:
I like that suggestion, @philpep. Do you want to make the change, or shall I?

Reply:
I'm affraid I can't because I'm not maintainer of anosql and github doesn't support yet multiline suggestions :)

Reply:
I'll take care of it then, @philpep. Thanks for the input!